power.go

```diff
@@ -0,0 +1,14 @@
+package astilectron
+
+// Power event names
+const (
+	PowerEventSuspend             = "power.event.suspend"
```

Comment:
Can you prefix events with `EventNamePower` instead of `PowerEvent` ?

Reply:
I was initially going for `EventNamePower`, but it feels a bit clunky. Anyway, I'll change them to `EventNamePower`.

Reply:
Done